get-artifacts/get_artifacts.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
```

Comment:
Gotta say, I'm not really a fan of this being in bash...

But if you think bash is the best way to do this, at the very least this should have shellcheck run on it.

One immediate issue I can spot is that I think this should have `set -eo pipefail` at the top.

Reply:
shellcheck added to both big scripts in this repo.   had it on diem/diem, added it over before I saw this comment.  have it in the ide as well.   renamed the create-artifacts.yml to ci-test.yaml to add a shellcheck section (still produces artifacts).